rpm/pcs.spec.in

```diff
@@ -36,8 +44,8 @@ Summary: Pacemaker/Corosync Configuration System
 
 %global pcs_bundled_dir @pcs_bundled_dir@
 
-%global required_pacemaker_version 2.1.0
-
+%global required_pacemaker_min_version 2.1.0
+%global required_pacemaker_max_version 3.0.0
```

Comment:
This is not a good name. It was supposed to mark a version of pacemaker which is too new for pcs to work with. However, the variable name suggests it is a version of pacemaker that pcs still works with.